WooCommerce.com login - move ToS above actions

mcliwanow opened the pull request:
#### Proposed Changes

ToS texts were moved above action buttons for both "regular" and social login.

Additionally, @wagveloso pointed out that the form should be narrower. Form width and other spacings were updated accordingly to designs in Figma BRQO7Z06r6AS4HOg86BuoN-fi-3043%3A7869 (all styles are nested inside `.woo` class - commit https://github.com/Automattic/wp-calypso/pull/66117/commits/3d27d85f58e0386b700369f1f6e1525eaa5554f8). 

#### Testing Instructions

<!--
Add as many details as possible to help others reproduce the issue and test the fix.
"Before / After" screenshots can also be very helpful when the change is visual.
-->

- On WooCommerce.test click login
- Replace _https://wordpress.com/_ with _http://calypso.localhost:3000/_ in the url (make sure that you followed the steps from the Calypo's [Getting Started](https://github.com/Automattic/wp-calypso/blob/trunk/README.md#getting-started) section of the README file)
- Go through the login flow and verify that the Terms of Service are displayed above the action buttons
- Go through login and registration screen and verify that width of login/registration form is correct
- Please test around this issue

#### Screenshots
![image](https://user-images.githubusercontent.com/4765119/182796376-625b646d-2d2e-4821-80d8-5862afe86ece.png)
![image](https://user-images.githubusercontent.com/4765119/182796439-4cd28439-9cb9-4286-b98d-24a30122da47.png)

mcliwanow committed:
WooCommerce.com login - move ToS above actions

ToS texts were moved above action buttons for both "regular" and social login.

tyxla reviewed (commented):
Have we tested with the other scenarios that use oauth? Those visual changes might be inappropriate for Jetpack Cloud or Crowdsignal (or any other one really)

mcliwanow commented:
@tyxla , the changes are nested inside `.woo` class. They should affect only WooCommerce.com login. I checked the default WP.com login and it wasn't affected by the changes.

tyxla reviewed (approved):
No reservations from the frameworks perspective 👍

mcliwanow committed:
Update styles for Woo login

- Login card and form are narrower on desktop
- Social connect - updated icon sizes so they look more natural (WordPress icon has some padding in SVG so it looked smaller than Google/Apple icons)
- Update spacing of some elements

andfinally reviewed (commented):
The Google and Apple buttons are squished for me on FF. In the Figma, they have a width of 187px each. It might be worth decreasing the side padding of `.woo login__social` to 80px.

<img width="320" alt="image" src="https://user-images.githubusercontent.com/1647564/182877790-c3097c96-16e9-4618-962f-d7265676abf0.png">

andfinally reviewed (commented):
Looks good to me, apart from the issue with the social buttons, and this small point.

"Terms of Service" is 14px on the signup form, should be 12px like the rest of `.signup-form__terms-of-service-link`.

<img width="425" alt="image" src="https://user-images.githubusercontent.com/1647564/182883805-57387c2a-44be-4169-81d2-72138ab94440.png">

KokkieH reviewed (approved):
Tests as described, and can't spot any issues aside from the "Terms of service" font size issue on the signup page that Gareth mentioned.

The social login buttons appear fine for me in Firefox.
![Log_In_—_WordPress_com](https://user-images.githubusercontent.com/11873759/183005613-f276c90d-c598-4ac4-aa43-9f4c15e29de7.png)

mcliwanow committed:
Woo Login form - increase width

Additionally, fix font-size of link nested inside .signup-form__terms-of-service-link.

mcliwanow commented:
@andfinally , @KokkieH , I was able to reproduce the issue with social login buttons on FF. Decreasing padding to 85px seems to be more than enough.
This, along with fixing font size of Terms of Service link, is fixed in https://github.com/Automattic/wp-calypso/pull/66117/commits/3f3b2757faba0afe0003060f0b1ba89d8e39ba6e .
